### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by mzaidikhlas** [asadm/urduscript#9]
I think we should start documenting UrduScript at this point so that we can lay out the base and future contributions can be accommodated accordingly. I really like [p5.js ](https://p5js.org/reference/) & [Unity](https://docs.unity3d.com/ScriptReference/MonoBehaviour.Update.html)'s documentation because they are crafted for beginners. 

There are a lot of tools like [YUIDoc](http://yui.github.io/yuidoc/), [JSDoc](http://usejsdoc.org/) and [ESDoc](http://usejsdoc.org/) that we can use.

P.S. [p5.js ](https://p5js.org/reference/) uses  [YUIDoc](http://yui.github.io/yuidoc/).

**Comment by asadm** [comment]
Yes. I am supposed to explore making docs this weekend. If you have time and have used YUIDoc, PR is most welcome!

**Comment by mzaidikhlas** [comment]
I haven't used it yet but I'll look into it.
